lib/HLSL/DxilContainerReflection.cpp

```diff
@@ -2389,10 +2360,13 @@ HRESULT DxilModuleReflection::_GetResourceBindingDesc(UINT ResourceIndex,
   _Out_ D3D12_SHADER_INPUT_BIND_DESC *pDesc, PublicAPI api) {
   IFRBOOL(pDesc != nullptr, E_INVALIDARG);
   IFRBOOL(ResourceIndex < m_Resources.size(), E_INVALIDARG);
+#ifdef _WIN32
```

Comment:
splitting a conditional like this is pretty gross... Is there no way to make the `if` statement valid on Linux?

If not, you could make the if return after the memcpy, which would eliminate the need for an else block.